SPEC.md

```diff
@@ -106,10 +72,29 @@ Errors are indicated by a non-zero return code and the following JSON being prin
 ```
 
 Error codes 0-99 are reserved for well-known errors (to be defined later).
-Values of 100+ can be freely used for plugin specific errors. 
+Values of 100+ can be freely used for plugin specific errors.
 
 In addition, stderr can be used for unstructured output such as logs.
 
+### DEL command
+
+The DEL command is executed when the container needs to be deleted (leave) from a network.
+It is typically done as part of the container teardown.
+
+#### Parameters passed via environment variables
+- `CNI_COMMAND`: `DEL`
+- `CNI_CONTAINERID`: Container ID. This must be the same Container ID that was passed in during the ADD command. If no Container ID was passed into the ADD command, no ID must be passed in during the DEL command.
+- `CNI_NETNS`: Path to network namespace file representing network namespace from which the container is to be deleted, i.e. `/proc/[pid]/ns/net` or a bind-mount/link to it.
+- `CNI_PATH`: Colon-separated list of paths to search for CNI plugin executables.
+
+#### State streamed via standard input
+This is the JSON value that was returned in the `state` field of the successful result JSON document during the ADD command.
```

Comment:
We can wrap it in an object if we expect to add other stuff to it down the road. Otherwise it's just a dictionary with a single element for no reason.